live-build/variants/internal-dcenter/ansible/playbook.yml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- hosts: all
+  connection: chroot
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: /usr/bin/python2
```

Comment:
For my own education, why do we need to use the deprecated Python 2 runtime as the Ansible Python interpreter? We use Python 3 in all the other Ansible playbooks in this repository, so it would make more sense to me to be consistent and to use a non-deprecated version of Python. If there's a special reason we need to be using Python 2, perhaps a comment might clarify.

Reply:
I asked for a comment as well.. python2 is needed here, so that when "pip" is used in the role, it installs the python2 module rather than the python3 module (assuming I understand correctly based on my prior discussions with george).

If we can use the "apt" packages rather than "pip" packages, this might no longer be necessary (and doing that would be better, imo).